frontend/website/README.md

```diff
@@ -0,0 +1,27 @@
+### How to build the website
+
+First get the code:
```

Comment:
Just tried this - and failed because don't have pandoc

Need to add step for `brew install pandoc`

Reply:
that also makes me realize I have to update the CI to handle the new build script, thanks